client/e2e/src/pages/hugging-face-model-download.po.ts

```diff
@@ -0,0 +1,215 @@
+import { browser, by, element, ElementArrayFinder, ElementFinder, protractor } from 'protractor';
+
+import { TestUtils } from './test-utils';
+import { ModelFile } from './model-file';
+import { OMZModelPrecisionEnum } from '../../../src/app/modules/model-manager/components/model-downloader-table/model-downloader-table.component';
+
+export const filterGroupNames = ['task', 'library', 'model-type', 'language', 'license'];
+type FilterGroupName = typeof filterGroupNames[number];
```

Comment:
As far as I understand, `FilterGroupName` will be a `string` type anyway. If we want to limit to only types from an array `filterGroupNames` , then we can do this:
`type FilterGroupName = 'task' | 'library' | 'model-type' | 'language' | 'license'`

Reply:
Yes, this was a previous iteration. Yet with this approach we would not be able to iterate over `type FilterGroupName`.